editors/sc-ide/SuperColliderIDE.desktop

```diff
@@ -5,6 +5,5 @@ Exec=scide %F
 Icon=sc_ide
 Type=Application
 Terminal=false
-Patterns=*.scd;*.sc;*.schelp
```

Comment:
No this is handled by the mime xml file.
Moreover having the `Patterns` tag in the desktop file is invalid as it is only allowed in desktop files of type `MimeType` not `Application` (and even type `MimeType` is deprecated in favor of mime xml files).